tailwind.config.ts

```diff
@@ -15,6 +15,12 @@ export default {
         sans: ['Montserrat', ...defaultTheme.fontFamily.sans],
       },
     },
+    screens: {
+      sm: '640px',
+      md: '768px',
+      lg: '1024px',
+      xl: '1280px',
+    },
```

Comment:
I think we might not need this since the default screen in tailwind is the same:  https://github.com/tailwindlabs/tailwindcss/blob/66929501c192be875dd133050a1fa8e54675d34b/stubs/config.full.js#L815

hummm, or we can add a bit more size, the following is the inspector sizing options
```
Mobile S: 320px
Mobile M: 370px
Mobile L: 425px

Tablet: 768px

Laptop: 1024px
Laptop L: 1440px

4K: 2560px
```